client/web/app/elements/lecture/view.html

```diff
@@ -40,6 +40,21 @@
             padding: 0px;
             background: black;
         }
+        paper-card.rate { @apply(--layout-horizontal); }
+        .card-img {
+            padding: 20px;
+        }
+        .rate-content {
+            @apply(--layout-flex);
+            float: left;
+            cursor: pointer;
+        }
+        .rate-header { @apply(--paper-font-headline); }
+        .rate-name { color: var(--paper-grey-600); margin: 10px 0; }
```

Comment:
jestem zwolennimkiem żeby formatować to w tradycyjny sposób, czyli 

x {
   color: 
}

inaczej jest mniej czytelnie. Ale to olać :)